AutoRest/Generators/Ruby/Ruby/Templates/MethodTemplate.cshtml

```diff
@@ -95,11 +95,7 @@ def @(Model.Name)_async(@(Model.MethodParameterDeclaration))
       if (parameter.IsConstant)
       {
   @:@(parameter.Name) = @(parameter.DefaultValue)
-      }
-      else
-      {
-  @:@(parameter.Type.ValidateType(Model.Scope, parameter.Name))

```

Comment:
Validation method must be added back to the models and add a test that takes care for this